### PR TITLE
feat: add skipRowOnError for raw line protocol files

### LINKED ISSUE #415: Add --skipRowOnError for raw Line Protocol files
FR: https://github.com/influxdata/feature-requests/issues/349

### PR DESCRIPTION
**Pull request by candrewlee14**
Closes #415

### CODE REVIEW

File: pkg/csv2lp/points.go [hunk 1]
```diff
@@ -0,0 +1,2895 @@
+// Original source from models/points.go in "github.com/influxdata/influxdb/"
```
Comment (jeffreyssmith2nd) [inline pkg/csv2lp/points.go:1]: Is this code modified at all from what is in the influxes repo?

File: pkg/csv2lp/lp_reader.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	lineReader := NewLineReader(reader)
+	lineReader.LineNumber = 1 // start counting from 1
+	return &LineProtocolFilterReader{
+		lineReader: lineReader,
+	}
+}
+
+func (state *LineProtocolFilterReader) Read(b []byte) (int, error) {
+	for {
+		// buf := make([]byte, len(b))
```
[the remark this answers is not in the document]
Comment (candrewlee14) [reply pkg/csv2lp/lp_reader.go:27]: The for loop gets continued when the line protocol is bad, so it runs until we good good line protocol or EOF.

File: pkg/csv2lp/points.go (hunk 1 above)
Comment (candrewlee14) [reply pkg/csv2lp/points.go:1]: I believe not at all, just contains points.go and 2 or 3 other smaller files that had utility functions that were used.

File: pkg/csv2lp/lp_reader.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (jeffreyssmith2nd) [reply pkg/csv2lp/lp_reader.go:27]: Sorry, wasn't clear I was referencing the commented out code

**Review by jeffreyssmith2nd — APPROVED** [review]

**Pull request merged**